### PR TITLE
Hymnal App Database one-time setups

### PR DESCRIPTION
**Pull request by brandonxia01**
Set up some stuff for database layer code and created Database table queries which only need to be run once (but are idempotent)

### CODE REVIEW

**Review by brotatotes — COMMENTED** [review]
Overall this looks awesome. I left some relevant comments/questions in the design doc, so I won't duplicate them here.

File: src/db/DbModels.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  LYRIC_TYPE_CHORUS,
+  LYRIC_TYPE_BRIDGE,
+}
+
+// Data object for Lyrics table
+interface DbLyric {
+  songId: typeof uuidv4;
+  lyricType: LyricType;
+  verseNumber: number;
+  presentationOrder: number;
```
Comment (brotatotes) [inline src/db/DbModels.tsx:38]: I think can be moved to DbSong

Comment (brotatotes) [inline src/db/DbModels.tsx:38]: I think can be moved to DbSong

**Review by brotatotes — CHANGES_REQUESTED** [review]

File: src/db/DbModels.tsx (hunk 1 above)
Comment (brandonxia01) [reply src/db/DbModels.tsx]: done. dbs updated as well

**Review by brotatotes — APPROVED** [review]
epic!!!

**Pull request merged**